### PR TITLE
fix(iporder modale): tooltip position and minor improvement

### PR DESCRIPTION
**Pull request by fchabrolle**
## tooltip position fix and minor improvement using ui-kit

ref MANAGER-1273

### CODE REVIEW

File: client/app/ip/ip/order/DEDICATED.html [hunk 1]
```diff
@@ -5,35 +5,31 @@
...
-                data-ng-options="r as ((r > 1 ? 'ip_order_ip_count_other' : 'ip_order_ip_count_1') | translate:{ t0: r }) for r in orderableIp.ipv4BlockSizesAll">
-            <option value=""
-                    data-ng-disabled="true"
-                    data-translate="global_select">
-            </option>
-        </select>
+        <oui-select id="blockSizeSelect"
+                    name="blockSizeSelect"
+                    required
+                    placeholder="{{ 'global_select' | translate }}"
```
Comment (marie-j) [inline client/app/ip/ip/order/DEDICATED.html]: Missing `data-`

File: client/app/ip/ip/order/DEDICATED.html [hunk 2]
```diff
@@ -5,35 +5,31 @@
...
-                data-ng-required="true">
-            <option value=""
-                    data-ng-disabled="true"
-                    data-translate="global_select">
-            </option>
-        </select>
+        <oui-select id="countrySelect"
+                    name="countrySelect"
+                    required
+                    placeholder="{{ 'global_select' | translate }}"
```
Comment (marie-j) [inline client/app/ip/ip/order/DEDICATED.html]: Same here

**Comment by antleblanc** [comment]
Due to #643, could you please resolve the conflict?

File: client/app/ip/ip/order/PCC.html [hunk 3]
```diff
@@ -5,121 +5,98 @@
...
-                  data-ng-required="true"
-                  name="description"
-                  style="resize: none;"
-                  data-ng-model="model.params.description">
-        </textarea>
-        <i class="fa fa-info-circle"
-           data-tooltip-placement="left"
-           data-uib-tooltip="{{ 'ip_order_description_help' | translate }}">
-        </i>
+        <oui-field label="{{ 'ip_order_description' | translate }}"
```
Comment (cbourgois) [inline client/app/ip/ip/order/PCC.html]: `data-label` , `data-label-popover`

File: client/app/ip/ip/order/ip-ip-order.controller.js [hunk 4]
```diff
@@ -152,12 +173,12 @@ angular.module('Module.ip.controllers').controller('IpOrderCtrl', ($scope, $root
       case 'PCC':
         return (
           $scope.model.params.size
-            && $scope.model.params.country
-            && $scope.model.params.networkName
-            && /^[a-zA-Z]+\w+$/.test($scope.model.params.networkName)
-            && $scope.model.params.estimatedClientsNumber
-            && $scope.model.params.description
-            && $scope.model.params.usage
+                    && $scope.model.params.country
```
Comment (antleblanc) [inline client/app/ip/ip/order/ip-ip-order.controller.js]: I suspect that you broke indentation:)

**Comment by jleveugle** [comment]
Hello @fchabrolle,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**